server/src/main/java/io/crate/replication/logical/action/PublicationsStateAction.java

```diff
@@ -163,7 +176,17 @@ protected ClusterBlockException checkBlock(Request request,
         }
 
         @VisibleForTesting
-        static List<RelationName> resolveRelationsNames(Publication publication, Schemas schemas) {
+        static List<RelationName> resolveRelationsNames(Publication publication, String publicationName, Schemas schemas, UserLookup userLookup) {
+            var user = userLookup.findUser(publication.owner());
+            if (user == null) {
+                LOGGER.warn(
+                    "Tables of a publication '{}' won't be replicated as the user '{}' owning the publication is not found.",
+                    publicationName,
+                    publication.owner()
+                );
```

Comment:
If we will keep this behaviour it must result in an exception in order to stop the replication on the subscriber side. Otherwise it may retry forever which seems not to make sense. But we should first clarify which behaviour is expected.

Reply:
I checked PG behavior (first comment), abandoned pub/ or sub is not possible - so I will throw an Exception here